### DISCUSSION TITLE
One Result Displayed by Allure Despite Running Pytest in a Loop in jenkins

### DISCUSSION THREAD

**Discussion by assiakhateeb** [allure-framework/allure2#1985] [TestOps]
I'm running a Jenkins job that executes pytest in a loop. Here's the code snippet I'm using:
<br>
```stages {
  stage("tests") {
    steps {
      withAllureUpload(serverId: 'allure-testops', projectId: '1', results: [[path: 'allure-results']]) {
        sh """
        export PYTEST_ADDOPTS="--alluredir=$WORKSPACE/allure-results"
        pytest --loop=10 test_file.py 
        """
      }
    }
  }
}
```
<br>
However, when I check the test results in Allure TestOps, I only see the results from one run, not all the runs in the loop. Do you have any suggestions on how to fix this issue?
<br>
Note: After researching, it has been found that duplicate tests with the same name are recognized as a single test. However, it is unclear whether changing the test name will resolve this issue.

**Comment by cheshi-mantu** [comment]
@assiakhateeb 
Hi Assia,
these test results you get, are considered as retries for the initial attempt and you'll find them in the Retries section of a launch which is 100% expected correct behaviour – you run same tests in same environment which is a retry for such test.

<img width="1289" alt="image" src="https://github.com/allure-framework/allure2/assets/8607348/37636290-cc26-4323-a602-0335a571f367">


<img width="1291" alt="image" src="https://github.com/allure-framework/allure2/assets/8607348/188b81ec-5ba5-45c0-b09e-e2f699d4753e">

**Reply by assiakhateeb** [reply]
@cheshi-mantu Thanks!!
